extensions/amp-lightbox/1.0/lightbox.js

```diff
@@ -164,15 +163,15 @@ function LightboxWithRef(
   return (
     mounted && (
       <ContainWrapper
-        id={id}
         ref={(r) => {
           lightboxRef.current = r;
         }}
-        style={{visibility: 'hidden'}}
         size={true}
         layout={true}
         paint={true}
-        className={classes.lightboxContainWrapper}
+        part="lightbox"
+        wrapperClassName={`${classes.defaultStyles} ${classes.wrapper}`}
+        wrapperStyle={{visibility: 'hidden'}}
```

Comment:
FYI: this is not great. But I guess we will fix in a followup PRs. Ideally we always set "final" values on the rendering level and modify whatever is needed for animation during the animation side effect.

Reply:
Moved to jss wrapper class - verified this is overridden by inline styles applied during animations.

Reply:
Still seeing it here. But what I actually want is a little different:

1. No `wrapperStyle={{visibility: hidden}}`.
2. No `visibility:hidden` (or opposite, only `visibility:visible`) in JSS.
3. All animation nuances are applied within the corresponding `useLayoutEffect`. E.g. the opening animation starts by setting style to `visibility:hidden` and finishes by setting it to `visible`.

Basically, if we disable animation effect - the styles are fully correct from the start.

Reply:
Done